tests/extensions/test_files_store.py

```diff
@@ -70,6 +73,30 @@ def test_spider_closed_even(caplog):
         ]
 
 
+def test_spider_closed_no_data(caplog):
+    spider = spider_with_files_store('no_data')
+
+    extension = FilesStore.from_crawler(spider.crawler)
+    with caplog.at_level(logging.INFO):
+        extension.spider_closed(spider, 'finished')
+        assert [record.message for record in caplog.records] == [
+            '+----------------------------- DATA DIRECTORY -----------------------------+',
+            '|                                                                          |',
+            '| SOMETHING FAILED AND NO DATA WAS DOWNLOADED. CHECK THE LOGS FOR DETAILS  |',
+            '|                                                                          |',
+            '+--------------------------------------------------------------------------+',
+        ]
+
+
+def test_spider_closed_fails(caplog):
+    spider = spider_with_files_store('failed')
```

Comment:
Match test name to reason.

Use tmpdir since test should still pass even if FILES_STORE exists.

```suggestion
def test_spider_closed_failed(tmpdir, caplog):
    spider = spider_with_files_store(tmpdir)
```